snapcloud_daemon

```diff
@@ -18,7 +18,7 @@ fi
 NAME=snapcloud
 DESC="SnapCloud service"
 PATH=/home/cloud/snapCloud
-DAEMON=/home/cloud/snapCloud/start.sh
+DAEMON=/home/cloud/snapCloud/bin/start.sh
 
```

Comment:
This is not working when calling `/etc/init.d/snapcloud-daemon start`.

I'll leave `start.sh` in the root as it was, I see no problem in having the main start script in the main dir.